### ISSUE TITLE
Accessing `document.addEventListener` breaks lenta.ru

### ISSUE THREAD

**Issue by eight04** [Tampermonkey/tampermonkey#722]
## Specifications

- Chromium: Version 77.0.3815.2 (Official Build) canary-dcheck (32-bit)
- TM: 4.8.41
- OS: Windows 7 x64

## Script

```js
// ==UserScript==
// @name Test
// @version 5.0.3
// @namespace eight04.blogspot.com
// @include *
// @grant GM_addStyle
// @run-at document-start
// ==/UserScript==

console.log(document.addEventListener);
```

Install the script and visit https://lenta.ru/ or https://www.gazeta.ru/. The page becomes empty when the script is activated.

It doesn't happen with Violentmonkey so I'm not really sure what is the root cause.

**Comment by Almashina** [comment]
Hello!
I just did like You said, disabled all scripts, activated just one test script, and yes - pages lenta.ru, gazeta.ru, rambler.ru becomes empty.
FF56.0.2 + tampermonkey

**Comment by derjanb** [comment]
Should be fixed at TM BETA 4.9.5971

Thanks for reporting.

**Comment by Almashina** [comment]
Thank You for Your work!
